module/metrics.go

```diff
@@ -159,17 +159,13 @@ type ConsensusMetrics interface {
 }
 
 type VerificationMetrics interface {
-	// Finder Engine
-	//
+	// TODO: remove this event handlers once we have new architecture in place.
```

Comment:
We plan to _group these methods_ into smaller child interfaces each for a separate engine: https://github.com/dapperlabs/flow-go/issues/5534